meshroom/nodes/aliceVision/ExportAnimatedCamera.py

```diff
@@ -117,9 +117,10 @@ class ExportAnimatedCamera(desc.AVCommandLineNode):
             name="outputImages",
             label="Undistorted Images",
             description="Output undistorted images.",
-            value=desc.Node.internalFolder + "undistort/" + "<INTRINSIC_ID>_<FILESTEM>.{undistortedImageTypeValue}",
+            value=lambda attr: desc.Node.internalFolder + "undistort/" + "<INTRINSIC_ID>_<FILESTEM>.{undistortedImageTypeValue}" if attr.node.exportUndistortedImages.value else "",
```

Comment:
adding "enabled" should be enough, we don't need to change the "value", it can stay static.